app/Domain/Mail/MailNotSend.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Domain\Mail;
+
+final class MailNotSend extends \DomainException
```

Comment:
Maybe add the ID of the integration for which it failed?

Reply:
While adding the id is difficult because i don't have an integration in that context (nor do I want to add it because not every mail is necessarly related to an integration), I now add the variables in the exception.
This should provide extra logging.